src/test/java/com/programmers/ticketparis/repository/performance/PerformanceRepositoryTest.java

```diff
@@ -0,0 +1,98 @@
+package com.programmers.ticketparis.repository.performance;
+
+import com.programmers.ticketparis.domain.performance.Performance;
+import org.junit.jupiter.api.*;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import java.time.LocalDate;
+
+import static com.programmers.ticketparis.domain.performance.Category.MUSICAL;
+
+@SpringBootTest
+@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
+class PerformanceRepositoryTest {
+
+    @Autowired
+    private PerformanceRepository performanceRepository;
+
+    @Order(1)
+    @Test
+    @DisplayName("😀 정상적으로 공연이 레포지토리에 저장되는지 테스트")
```

Comment:
```suggestion
    @DisplayName("공연을 레포지토리에 저장할수 있다.")
```

이 귀여운 😀이모지는 컨벤션 인가요?

Reply:
처음에 팀원들에게 "테스트 코드에 이모지 넣으면 리뷰에서 말할꺼야?"라고 말했던 것 같아요. 🥲🥲 그때 아무말이 없던 걸로 기억합니다. 그래서 테스트 코드에다가 이모지를 넣었는데, 리뷰에서 이모지 넣은게 지적 받았습니다.. 또르륵 😅😅
팀 컨벤션으로 인해 수정된 MybatisPerformanceRepositoryTest에는 이모지를 넣지 않았습니다.